jet_fuel/config/initializers/session_store.rb

```diff
@@ -0,0 +1,3 @@
+# Be sure to restart your server when you modify this file.
+
+Rails.application.config.session_store :cookie_store, key: '_jet_fuel_session'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.